Dockerfile

```diff
@@ -54,6 +54,8 @@ RUN sed -i "s/listen [0-9]*;/listen ${CONTAINER_PORT};/" $CONF_NGINX_SITE && \
     sed -i "s/}/\n    font\/woff                             woff;&/" /etc/nginx/mime.types && \
     sed -i "s/}/\n    font\/woff2                            woff2;\n&/g" /etc/nginx/mime.types
 
+RUN /bin/bash -e /scripts/set_permissions.sh
```

Comment:
does this need to be its own distinct layer? is it possible to combine with another?

Reply:
No it does not, shall I combine it with the layer above this?

Reply:
Yes please!

Reply:
Done